example/App.js

```diff
@@ -8,37 +8,37 @@ import CreatePortalExample from './createPortalExample';
 import SVGExample from './SVGExample';
 import LazySuspenseExample from './lazySuspenseExample';
 
-export default function App () {
+export default function App() {
   return (
     <div>
-      <div className="wrapper">
+      <div className='wrapper'>
```

Comment:
Most commonly the prop values are kept `double quotes` and the js part is kept single quotes like this

```js
const a = 'hello';
return (
  <div className="wrapper" style={{ backgroundColor: 'black' }}>
    {a}
  </div>
);
```

@s-yadav what do you think?